ui/types/globals.d.ts

```diff
@@ -28,14 +28,21 @@ type GlobalQuasarIconMapFn = (
   iconName: string
 ) => { icon: string } | { cls: string; content?: string } | void;
 
+interface GlobalQuasarBex {
```

Comment:
I'd put this into "@quasar/app" types, keeping the indirection level which is in place for "cordova" and "electron" to avoid creating problems for Vue CLI users.
The file used for those is "quasar/app/types/globals.d.ts".
It's mostly a mattery of consistency, your types right now doesn't break anything (but could in the future if you have to reference external types provided by "@quasar/app").
Your choice about actually moving it there.
You could then put it directly into "quasar/app/types/globals.d.ts" or into a standalone "bex.d.ts" (and import it into global declaration files).